tests/llm/test_llm_client.py

```diff
@@ -26,6 +27,7 @@
             finish_reason="stop",
             index=0,
             message=ChatCompletionMessage(content="This is a test!", role="assistant", function_call=None),
+            logprobs=None,
```

Comment:
unrelated change ?

Reply:
IIRC pydantic was complaining while this parameter was missing...